### ISSUE TITLE
cannot import name 'cred'

### ISSUE THREAD

**Issue by somedevreally** [pictuga/morss#42]
after todays update, morss stopped with following error.
am running on docker

```
aceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/lib/python3.8/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/lib/python3.8/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/lib/python3.8/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/usr/lib/python3.8/site-packages/morss/__init__.py", line 3, in <module>
    from .wsgi import application
  File "/usr/lib/python3.8/site-packages/morss/wsgi.py", line 20, in <module>
    from . import cred

ImportError: cannot import name 'cred' from partially initialized module 'morss' (most likely due to a circular import) (/usr/lib/python3.8/site-packages/morss/__init__.py)
[2020-08-24 15:12:28 +0000] [8] [INFO] Worker exiting (pid: 8)
[2020-08-24 15:12:28 +0000] [9] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/lib/python3.8/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/lib/python3.8/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/lib/python3.8/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/usr/lib/python3.8/site-packages/morss/__init__.py", line 3, in <module>
    from .wsgi import application
  File "/usr/lib/python3.8/site-packages/morss/wsgi.py", line 20, in <module>
    from . import cred

ImportError: cannot import name 'cred' from partially initialized module 'morss' (most likely due to a circular import) (/usr/lib/python3.8/site-packages/morss/__init__.py)

[2020-08-24 15:12:28 +0000] [9] [INFO] Worker exiting (pid: 9)
/usr/lib/python3.8/os.py:1023: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
  return io.open(fd, *args, **kwargs)
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 210, in run
    self.sleep()
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 360, in sleep
    ready = select.select([self.PIPE[0]], [], [], 1.0)
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 245, in handle_chld
    self.reap_workers()
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/bin/gunicorn", line 11, in <module>
    load_entry_point('gunicorn==19.9.0', 'console_scripts', 'gunicorn')()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 61, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/base.py", line 223, in run
    super(Application, self).run()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/base.py", line 72, in run
    Arbiter(self).run()
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 232, in run
    self.halt(reason=inst.reason, exit_status=inst.exit_status)
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 345, in halt
    self.stop()
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 393, in stop
    time.sleep(0.1)
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 245, in handle_chld
    self.reap_workers()
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
[2020-08-24 15:12:28 +0000] [10] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/lib/python3.8/site-packages/gunicorn/workers/base.py", line 129, in init_process
    self.load_wsgi()
  File "/usr/lib/python3.8/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
    return self.load_wsgiapp()
  File "/usr/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/lib/python3.8/site-packages/gunicorn/util.py", line 350, in import_app
    __import__(module)
  File "/usr/lib/python3.8/site-packages/morss/__init__.py", line 3, in <module>
    from .wsgi import application
  File "/usr/lib/python3.8/site-packages/morss/wsgi.py", line 20, in <module>
    from . import cred
ImportError: cannot import name 'cred' from partially initialized module 'morss' (most likely due to a circular import) (/usr/lib/python3.8/site-packages/morss/__init__.py)
[2020-08-24 15:12:28 +0000] [10] [INFO] Worker exiting (pid: 10)
```

**Comment by pictuga** [comment]
Opps, I forgot to push the latest commits that fix that. I'll push them within 24hrs

**Comment by pictuga** [comment]
Fixed in 75935114e4721a0aa3b7706675621931a7f8d8f0
